### PR TITLE
Add documentation for number of AWS API calls

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
closes https://github.com/elastic/beats/issues/15154

### CODE REVIEW

File: metricbeat/docs/modules/aws.asciidoc [hunk 1]
```diff
@@ -90,6 +90,73 @@ for a list of AWS services that publish metrics to CloudWatch.
...
+value is depends on the number of results.
+
+[float]
+=== `cloudwatch`
+|===
+| AWS API Name | AWS API Count | Frequency
+| IAM ListAccountAliases | 1 | Once on startup
+| STS GetCallerIdentity | 1 | Once on startup
+| EC2 DescribeRegions| 1 | Once on startup
+| Cloudwatch ListMetrics | Total Number of results / 500 | Per region per namespace per collection period
```
Comment (mtojek) [inline metricbeat/docs/modules/aws.asciidoc]: nit: sometimes it's safer not to give values here. I'm pretty sure the next person who will change the page size will also forget about this docs references. Maybe instead use "/ page size" ?

Comment (mtojek) [inline metricbeat/docs/modules/aws.asciidoc]: nit: number (lowercase)?

File: metricbeat/docs/modules/aws.asciidoc [hunk 2]
```diff
@@ -90,6 +90,73 @@ for a list of AWS services that publish metrics to CloudWatch.
...
+
+[float]
+=== `ec2`
+|===
+| AWS API Name | AWS API Count | Frequency
+| IAM ListAccountAliases | 1 | Once on startup
+| STS GetCallerIdentity | 1 | Once on startup
+| EC2 DescribeRegions| 1 | Once on startup
+| EC2 DescribeInstances | 1 | Per region per collection period
+| Cloudwatch ListMetrics | Total Number of results / 500 | Per region per collection period
```
Comment (mtojek) [inline metricbeat/docs/modules/aws.asciidoc]: nit: CloudWatch

File: metricbeat/docs/modules/aws.asciidoc (hunk 1 above)
Comment (kaiyan-sheng) [reply metricbeat/docs/modules/aws.asciidoc]: 500 and 100 here are actually from AWS API documentation. I will document that with the api links. Thanks!

**Pull request merged**